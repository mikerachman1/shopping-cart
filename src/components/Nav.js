import React from "react";
import { Link } from "react-router-dom";

const Nav = () => {
  return (
    <nav>
      <h3>ðŸ“¦ Shipper-Shopper ðŸ›’</h3>
      <ul className="nav-links">
        <Link to="/">
          <li className="nav-link">About</li>
        </Link>
        <Link to="/shop">
          <li className="nav-link">Shop</li>
        </Link>
      </ul>
    </nav>
  );
};

export default Nav;