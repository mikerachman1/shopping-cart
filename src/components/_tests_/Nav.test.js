import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

import Nav from "../Nav";

describe('Nav component', () => {
  it('renders navbar correctly', () => {
    const { container } = render(<Nav />, {wrapper: MemoryRouter});
    expect(container).toMatchSnapshot();
  });
});