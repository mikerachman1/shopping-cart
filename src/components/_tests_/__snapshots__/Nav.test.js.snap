// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders navbar correctly 1`] = `
<div>
  <nav>
    <h3>
      ðŸ“¦ Shipper-Shopper ðŸ›’
    </h3>
    <ul
      class="nav-links"
    >
      <a
        href="/about"
      >
        <li
          class="nav-link"
          name="About"
        >
          About
        </li>
      </a>
      <a
        href="/"
      >
        <li
          class="nav-link"
          data-testid="nav-shop-link"
          name="Shop"
        >
          Shop
        </li>
      </a>
      <a
        href="/cart"
      >
        <li
          class="nav-link"
          data-testid="nav-cart-link"
          name="Cart"
        >
          Cart
        </li>
      </a>
    </ul>
  </nav>
</div>
`;
